### DISCUSSION TITLE
Enable to Add email address

### DISCUSSION THREAD

**Discussion by Shalini-prasad** [community/community#57683] [New to GitHub]
Good Evening, 

My problem is that I wanted to add my college email address  to my personal Github account. But unable to do so.

Kindly help as soon as possible.

**Comment by Rimkomatic** [comment]
Hi!

I'd suggest you launch a ticket over to Github Education's Support Engineers regarding this matter. You can do so [here](https://support.github.com/contact/education).
please note sometimes the university restricts the email use

**Comment by airtower-luna** [comment]
Generally all that should be needed is to follow the instructions in the documentation: https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-email-preferences/adding-an-email-address-to-your-github-account

If you have any trouble with that it's hard to say what might be wrong without a detailed description of the problem.
